src/styles/sign-up-page.scss

```diff
@@ -342,6 +342,38 @@
     width: 133px;
   }
 
+  .saved-icon--small {
+    background-image: url('./../assets/icons/saved.png');
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: contain;
+    margin: auto;
+  }
+
+  /*
+    #Markers
+  */
+
+  .marker-point {
+    background-color: rgba(239, 197, 55, 0.7);
```

Comment:
Maybe use variables for color. It's easier to reuse them and you can give a descriptive name. For example:
`$yellow: rgba(239, 197, 55, .7)`

Also, is better to remove 0. Just with `.7` is ok.